spec/xcpretty/parser_spec.rb

```diff
@@ -164,6 +164,11 @@ module XCPretty
       @parser.parse(SAMPLE_OLD_SPECTA_FAILURE)
     end
 
+    it "parses ld bitcode errors" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.